packages/perseus-core/src/widgets/label-image/label-image-util.ts

```diff
@@ -0,0 +1,41 @@
+import type {
+    PerseusLabelImageMarker,
+    PerseusLabelImageWidgetOptions,
+} from "@khanacademy/perseus-core";
+
+/**
+ * For details on the individual options, see the
+ * PerseusLabelImageWidgetOptions type
+ */
+type LabelImagePublicWidgetOptions = {
+    choices: PerseusLabelImageWidgetOptions["choices"];
+    imageUrl: PerseusLabelImageWidgetOptions["imageUrl"];
+    imageAlt: PerseusLabelImageWidgetOptions["imageAlt"];
+    imageHeight: PerseusLabelImageWidgetOptions["imageHeight"];
+    imageWidth: PerseusLabelImageWidgetOptions["imageWidth"];
+    markers: ReadonlyArray<LabelImageMarkerPublicData>;
+    hideChoicesFromInstructions: boolean;
+    multipleAnswers: boolean;
+    static: boolean;
+};
+
+type LabelImageMarkerPublicData = Pick<
+    PerseusLabelImageMarker,
+    "x" | "y" | "label"
+>;
+
+export default function getLabelImagePublicWidgetOptions(
+    options: PerseusLabelImageWidgetOptions,
+): LabelImagePublicWidgetOptions {
+    return {
+        ...options,
+        markers: options.markers.map(getLabelImageMarkerPublicData),
+    };
+}
+
+function getLabelImageMarkerPublicData(
+    marker: PerseusLabelImageMarker,
+): LabelImageMarkerPublicData {
+    const {answers: _, ...publicData} = marker;
+    return publicData;
```

Comment:
So here, instead of building out the wanted object, you're using destructuring to pull out answers, is that correct? What does using the underscore do? Is that just to help signify that we aren't interested in the answers here? Looks like it's needed to satisfy ESLint.

Reply:
> What does using the underscore do? Is that just to help signify that we aren't interested in the answers here?

Correct!